core/templates/pages/story-editor-page/modal-templates/new-chapter-title-modal.controller.spec.ts

```diff
@@ -133,25 +134,21 @@ describe('Create New Chapter Modal Controller', function() {
       expect($scope.editableThumbnailBgColor).toEqual('abc');
     });
 
-  it('should call the StoryUpdateService to delete the story node', function() {
-    var storyUpdateSpy = spyOn(StoryUpdateService, 'deleteStoryNode');
-    $scope.cancel();
-    expect(storyUpdateSpy).toHaveBeenCalled();
-  });
+  it('should delete the story node when closing the modal',
+    function() {
+      var storyUpdateSpy = spyOn(StoryUpdateService, 'deleteStoryNode');
+      $scope.cancel();
+      expect(storyUpdateSpy).toHaveBeenCalled();
+    });
 
-  it('should call the StoryUpdateService to update the title', function() {
+  it('should update the title when changing chapter title', function() {
```

Comment:
```suggestion
  it('should update the title', function() {
```
Here and above the when part of the name seems redundant. In general avoid saying something like "should update X when changing X", the when I think is implied

Reply:
Done.